src/exco/util.py

```diff
@@ -136,3 +138,24 @@ def group_by(f: Callable[[T], T2], xs: List[T]) -> Dict[T2, List[T]]:
         key = f(x)
         ret[key].append(x)
     return dict(ret)
+
+
+def get_merged_cell(sheet: Worksheet, coordinates: str) -> Optional[MergedCellRange]:
```

Comment:
can you put in doc string this is less than trivial
"""Find the merge cell in the whole sheet that contains coordinates

Args: 

Returns:


...


"""

Reply:
added doc string in but for ease of checking I added in 
```
    """Find the merge cell in the whole sheet that contains coordinates
    
    Args:
        sheet (Worksheet): worksheet
        coordinates (str): coordinates of cell

    Returns:
        MergedCellRange if cell is a part of a merged cell, None if cell is not a merged cell
    """
```
and will push once I can get this circular import from the above comment fixed.